invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides

```diff
@@ -0,0 +1,33 @@
+/*******************************
+    User Variable Overrides
+*******************************/
+
+@import "@less/invenio_theme/theme/elements/list.overrides";
+
+/* Deposit license field */
+
+.item.deposit-drag-listitem {
+    margin-bottom: 5px;
+    &.hidden {
+      opacity: 0;
+    }
+    .drag-anchor {
+        cursor: move;
+    }
+}
```

Comment:
If some one wants to add a new style or class for deposit license files, it should be done here?

Reply:
I would say yes :) That way we organize the custom classes per field!